options.js

```diff
@@ -68,6 +44,10 @@ export default function Options() {
     });
   };
   this.setAlternateUrl = async function(url) {
-    return writeToStorage(alternateUrlKey, url);
+    return utils.writeToStorage(alternateUrlKey, url);
+  };
+
+  this.getSaveErrors = async function() {
+    return Promise.resolve(true);
   };
```

Comment:
Was this meant to always return true?

Reply:
Nope, definitely not. Good catch, thanks!